sidecar/pkg/sidecar/sidecar_utils.go

```diff
@@ -124,15 +123,21 @@ func NewSideCarController(kubeClient *kubernetes.Clientset, controllerClient *cl
 	}
 
 	tenantInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc: func(new interface{}) {
+			fmt.Println("new")
```

Comment:
Debug code? If not, then please make it a bit more descriptive.

Reply:
Yep

Reply:
Not removed @jiuker yet